_includes/work-for-us.html

```diff
@@ -54,7 +54,7 @@
             <li>3 weeks paid vacation</li><br>
             <li>Year end paid break</li><br>
             <li>Competitive medical and dental benefits fully funded by Greenfield</li><br>
-            <li>We bill clients weekly for a 4 day work week, so Fridays are devoted to personal skill development and exploration</li><br>
+            <li>We bill clients weekly for a 4 day work week, Fridays are devoted to personal skill development and exploration</li><br>
```

Comment:
Also, I pretty much have no idea how to properly use a semicolon except for to end a statement in javascript;